web/src/main/java/com/epam/rft/atsy/web/configuration/ExceptionMapperConfiguration.java

```diff
@@ -12,47 +12,68 @@
 import com.epam.rft.atsy.service.exception.passwordchange.PasswordNewMatchValidationException;
 import com.epam.rft.atsy.service.exception.passwordchange.PasswordOldMatchValidationException;
 import com.epam.rft.atsy.service.exception.passwordchange.PasswordUniqueValidationException;
-import com.epam.rft.atsy.web.mapper.Rule;
+import com.epam.rft.atsy.web.mapper.ExceptionMessagePair;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
 import java.util.Collections;
 import java.util.HashSet;
 import java.util.Set;
 
+/**
```

Comment:
Follow Javadoc conventions: write the first sentence as a short summary of the class, as Javadoc automatically places it in the class summary table (and index).